client/broadcast.go

```diff
@@ -142,3 +145,21 @@ func (ctx Context) BroadcastTxAsync(txBytes []byte) (*sdk.TxResponse, error) {
 
 	return sdk.NewResponseFormatBroadcastTx(res), err
 }
+
+// TxServiceBroadcast is a helper function to broadcast a Tx with the correct gRPC types
+// from the tx service. Calls `clientCtx.BroadcastTx` under the hood.
+func TxServiceBroadcast(grpcCtx context.Context, clientCtx Context, req *tx.BroadcastTxRequest) (*tx.BroadcastTxResponse, error) {
+	if req == nil || req.TxBytes == nil {
+		return nil, status.Error(codes.InvalidArgument, "invalid empty tx")
```

Comment:
I use here the error from gRPC. We could also use sdkerrors. Any opinions on which is better?

Reply:
maybe a good rule of thumb would be:
- use grpc errors in server implementations (like here), since they are directly returned to grpc
- use sdkerrors everywhere else